### PR TITLE
Documentation fixes (typos & more)

### PR DESCRIPTION
**Pull request by seeekr**

### CODE REVIEW

**Comment by seeekr** [comment]
(thanks for writing up that comparison between Beast and websocketpp, very helpful!)

**Comment by vinniefalco** [comment]
There are some changes in **design.qbk** which might generate one or two conflicts, I suggest you rebase this after *1.0.0-b17* is merged and inspect the diff. Thanks for your contribution!

**Comment by seeekr** [comment]
@vinniefalco Sure, will do!

**Comment by seeekr** [comment]
Rebased, didn't get any conflicts, changes look the same. Added more doc fixes.

File: doc/websocket.qbk [hunk 1]
```diff
@@ -227,7 +227,7 @@ void echo(beast::websocket::stream<boost::asio::ip::tcp::socket>& ws)
 {
     beast::streambuf sb;
     beast::websocket::opcode::value op;
-    ws.read(sb);
+    ws.read(op, sb);
```
Comment (vinniefalco) [inline doc/websocket.qbk]: Yikes! Good find!

Comment (seeekr) [inline doc/websocket.qbk]: Working my way through actually building something w/ Beast, so consuming all the existing docs in detail!

Comment (vinniefalco) [inline doc/websocket.qbk]: Glad to hear it! The docs are still a work in progress so if you need anything in particular explained, just open up an issue and I'll get to it promptly

**Comment by vinniefalco** [comment]
Sorry this took so long, it will be in **1.0.0-b22**

**Comment by vinniefalco** [comment]
This was merged, thanks!

**Pull request closed without merging**